test/app-spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import test from 'blue-tape';
```

Comment:
@Chris-Petty pretty epic - looks real nice to use. I *personally* use Tape and enzyme only because it's a simple, effective test harness setup. I think I'll have a look at Jest as I begin personal React Native apps!

One sec, I'll push the changes up.